.prettierrc.js

```diff
@@ -1,3 +1,10 @@
 module.exports = {
-  ...require('@grafana/toolkit/src/config/prettier.plugin.config.json'),
```

Comment:
what do you think of having a [.config](https://github.com/grafana/athena-datasource/tree/main/.config) folder and extending that rather than copying over the contents of the config folder into our config pages. That way its a bit more obvious what pieces we are directly extending in this repo and what pieces are coming from some sort of default configuration setting? Also if there ever is a way to like migrate to new configuration settings maybe it'll be easier?